src/.clang-format

```diff
@@ -27,7 +27,28 @@ IndentCaseLabels: true
 IndentWidth: 4
 IndentWrappedFunctionNames: true
 IndentPPDirectives: AfterHash
-IncludeBlocks: Preserve
+SortIncludes: CaseInsensitive
+IncludeBlocks: Regroup
+IncludeCategories:
+# Project includes
+  - Regex:  '^"[a-zA-Z\._-]+(/[a-zA-Z0-9\._-]+)*"$'
+    Priority: 1
+# Third party library includes
+  - Regex:  '<[[:alnum:].]+/[a-zA-Z0-9\._\/-]+>'
+    Priority: 3
+# Qt includes
+  - Regex:  '^<Q[a-zA-Z\._\/-]+>$'
```

Comment:
should include numbers and i think can drop dot/dash 
https://doc.qt.io/qt-6/classes.html